tests/scripts/webhook-config.yaml

```diff
@@ -1,52 +1,75 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-issuer
+  namespace: rook-ceph
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: rook-admission-controller-cert
+  namespace: rook-ceph
+spec:
+  dnsNames:
+    - ${SERVICE_NAME}
+    - ${SERVICE_NAME}.rook-ceph.svc
+    - ${SERVICE_NAME}.rook-ceph.svc.cluster.local
+  issuerRef:
+    kind: Issuer
+    name: selfsigned-issuer
+  secretName: rook-ceph-admission-controller
+---
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
-  name: ${WEBHOOK_CONFIG_NAME}
+  name: rook-ceph-webhook
   annotations:
-    cert-manager.io/inject-ca-from: ${NAMESPACE}/rook-admission-controller-cert
+    cert-manager.io/inject-ca-from: rook-ceph/rook-admission-controller-cert
 webhooks:
-  - name: cephcluster-wh-${SERVICE_NAME}-${NAMESPACE}.rook.io
+  - name: cephcluster-wh-rook-ceph-admission-controller-rook-ceph.rook.io
     rules:
-      - apiGroups:   ["ceph.rook.io"]
+      - apiGroups: ["ceph.rook.io"]
         apiVersions: ["v1"]
-        operations:  ["CREATE","UPDATE","DELETE"]
-        resources:   ["cephclusters"]
+        operations: ["CREATE", "UPDATE", "DELETE"]
+        resources: ["cephclusters"]
     clientConfig:
       service:
-        name: ${SERVICE_NAME}
-        namespace: ${NAMESPACE}
+        name: rook-ceph-admission-controller
+        namespace: rook-ceph
         path: /validate-ceph-rook-io-v1-cephcluster
-      caBundle: ${CA_BUNDLE}
-    admissionReviewVersions: ["v1", "v1beta1"]  # API server will try to use first version in the list which it supports.
+      caBundle: Cg==
```

Comment:
Where is this value coming from? It does not even seem to be base64. How can you know the content of the CM `extension-apiserver-authentication`?

Reply:
this is a placeholder, I have these uses in other projects ex: https://github.com/rook/nfs/blob/master/cluster/examples/kubernetes/nfs/webhook.yaml#L65 and also noticed the same in the demo we had last Thursday.

Reply:
Hum I'm not sure I understand how this placeholder is useful. It would be great of it was an env var and that we could use `envsubst` to replace it.

Reply:
How is this being replaced by an actual certificate? The doc does not mention anything about this. If this is deployed the way it is right now, it's not going to work right?

Reply:
@subhamkrai ?

Reply:
I'll dive more on this...but I see in the CI admission controller is working also tested  locally and I don't for now we need to change anything in docs. But I'll update

Reply:
> How is this being replaced by an actual certificate? The doc does not mention anything about this. If this is deployed the way it is right now, it's not going to work right?

I have tested the changes locally it is working also we run the webhook in CI and CI is also passing.

Reply:
to mention again, we don't actually need `caBundle` anymore...see this https://github.com/kubernetes-sigs/kubebuilder/pull/1699 I have added this link the GitHub commit message